src/odemis/util/img.py

```diff
@@ -39,6 +39,7 @@
 from matplotlib import cm
 
 FIT_TO_RGB = "fitrgb"
+RGB_AS_IS = "rgbasis"
```

Comment:
Please don't create duplicate constants. I'd suggest to import them from `acq.stream`. (an alternative would be to define them here, and import this module in stream. You could even put them in odemis.model.metadata, as TINT_*.